### PR TITLE
Add the community section from the wireframe

### LINKED ISSUE #80: Wireframe: Community section
Implement the "Community" section from the wireframe issue #57 

This should result in a file such as `template/homepage-something-band.tmpl` along with the associated `themes/ansible-community/sass/_homepage-something-band.scss` definition.

### PR DESCRIPTION
**Pull request by oraNod**
Fixes #80 

@leogallego @cybette I included the details from the issue here https://github.com/ansible-community/community-website/issues/80#issuecomment-1690494630

I think the mentioned "Devs & Contributors" content should go into the Contributor section of the homepage.

Also I put the meeting lists as a separate thing from the "social" icons. We could make it a single link to [the asking-questions-over-email section of the docs](https://docs.ansible.com/ansible/latest/community/communication.html?extIdCarryOver=true&sc_cid=701f2000001OH6uAAG#asking-questions-over-email)

However I felt like "Meeting Lists" doesn't really fit in the social category quite the same way. I thought I'd send the PR as is but can change it to an additional item on the row of icons. Thanks.

### CODE REVIEW

**Review by maxamillion — APPROVED** [review]
LGTM

**Review by GregSutcliffe — CHANGES_REQUESTED** [review]
Can we drop the mailing lists? It splits the audience with the forum and doesn't look as nice as the rest of the icons anyway. A link to the communication docs would be fine I think.

The icons are left-centred btw, which makes them look lopsided. But we can catch that in the style pass.

Otherwise LGTM

**Comment by cybette** [comment]
+1 on excluding the mailing lists, the rest LGTM!

**Review by GregSutcliffe — COMMENTED** [review]
Icons are centred now, yay! Just a question about the CoC, looking good!

File: templates/homepage-community-band.tmpl [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        <img class="section-icon" src="/assets/images/community.svg" />
+        <h2>{{ homepage.community.title }}</h2>
+      </div>
+    </div>
+    <div class="community-intro">
+      <p>{{ homepage.community.ansible_community }}</p>
+      <p>{{ homepage.community.get_involved }}</p>
+    </div>
+    <hr />
+    <div class="community-conduct">
```
[the remark this answers is not in the document]
Comment (oraNod) [reply templates/homepage-community-band.tmpl:14]: no, the button is too large but I was thinking we could get that in a styling round. I think it'd be fine to have the CoC in it's own band and that's a good suggestion.

[the remark this answers is not in the document]
Comment (GregSutcliffe) [reply templates/homepage-community-band.tmpl:14]: Opened #107 for this, feel free to drop it from here.

**Comment by oraNod** [comment]
@GregSutcliffe I've reworked the Code of Conduct part here and think it's better. I played around with it and it's not probably not worth creating a separate band just for that. A separate band will require yet another "Back to Top" link and maybe a new title and entry on the proposed tabbed navigation. I'd prefer to try and just keep the CoC discrete within the community section.

File: templates/homepage-community-band.tmpl (hunk 1 above)
[the remark this answers is not in the document]
Comment (oraNod) [reply templates/homepage-community-band.tmpl]: bah, I should have replied in this thread. https://github.com/ansible-community/community-website/pull/104#issuecomment-1708184339

my 2c is we make CoC more discrete but keep it in the community band.

**Comment by GregSutcliffe** [comment]
Hmm, I'm still unconvinced, the flow on that feels weird - maybe I'm just used to seeing it at the end of the page though. Looking for other opinions here, reviewers!

**Comment by oraNod** [comment]
@GregSutcliffe I think you're right. I like the idea of putting the code of conduct somewhere, maybe the footer like you said. But I'm sort of violating that "direct to the forum as much as possible" pattern we want to establish.

**Comment by GregSutcliffe** [comment]
I *do* think it needs to be present, I'm just not entirely sure this is the place (or perhaps it's just the style). But let's not hold this up over the bikeshedding, and continue in #107

**Pull request merged**